libraries/rush-lib/assets/rush-init-subspace/common-versions.json

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
Same with this file, is this the same as the `rush init` version?

Reply:
Changed to use the same files as `rush init`